pkg/os/util_linux.go

```diff
@@ -34,3 +35,23 @@ func writeToFileAsRoot(reason, content, filepath string, append bool) error {
 	}
 	return nil
 }
+
+func GetFirstExistentPath(paths []string) (string, error) {
+	readablePath := ""
```

Comment:
Or var readablePath String

Reply:
i think mine is more readable. People new to Go might not know "" is default.